#ifndef _DETECTION_SERVER_H
#define _DETECTION_SERVER_H

#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <sys/socket.h>
#include <netinet/in.h>
#include <arpa/inet.h>
#include <unistd.h>
#include <iostream>
#include <string.h>
#include <iostream>

using namespace std;

int string_to_int(string s);
int outof0(string str);

#endif